test/integration/local/processing/process-payment-requests.test.js

```diff
@@ -340,4 +341,58 @@ describe('process payment requests', () => {
     expect(holds.length).toBe(0)
     expect(mockSendMessage).not.toBeCalled()
   })
+
+  test('should process reduction request and create completed request if handleSchemeClosures equals true', async () => {
+    processingConfig.handleSchemeClosures = true
+
+    // first payment request
+    await savePaymentRequest(paymentRequest, true)
+
+    // add FRN to the closure DB
+    await addAgreementToClosureDB(paymentRequest.schemeId, paymentRequest.frn, paymentRequest.agreementNumber)
+
+    // second payment request
+    const recoveryPaymentRequest = createAdjustmentPaymentRequest(paymentRequest, RECOVERY)
+    const { paymentRequestId } = await saveSchedule(inProgressSchedule, recoveryPaymentRequest)
+
+    await processPaymentRequests()
+
+    const completedPaymentRequests = await db.completedPaymentRequest.findAll({
+      where: {
+        paymentRequestId,
+        frn: paymentRequest.frn,
+        marketingYear: paymentRequest.marketingYear,
+        schemeId: paymentRequest.schemeId,
+        ledger: AP,
+        value: -100
+      }
+    })
+    expect(completedPaymentRequests.length).toBe(1)
+  })
+
+  test('should not process manual ledger request if handleSchemeClosures equals true', async () => {
+    processingConfig.handleSchemeClosures = true
+
+    // first payment request
+    await savePaymentRequest(paymentRequest, true)
+
+    // add FRN to the closure DB
+    await addAgreementToClosureDB(paymentRequest.schemeId, paymentRequest.frn, paymentRequest.agreementNumber)
+
+    // second payment request
+    const recoveryPaymentRequest = createAdjustmentPaymentRequest(paymentRequest, RECOVERY)
+    await saveSchedule(inProgressSchedule, recoveryPaymentRequest)
+
+    await processPaymentRequests()
+
+    const holds = await db.hold.findAll({
+      where: {
+        frn: paymentRequest.frn,
+        closed: null
+      }
+    })
+
+    expect(holds.length).toBe(0)
+    expect(mockSendMessage).not.toBeCalled()
+  })
```

Comment:
No need for a reference to holds here as that's not what the behaviour can influence or what the test says it's testing.

Also similar to above suggest adding tests to cover that we still send to request editor if handleSchemeClosures is false or the date has not yet occured.

Reply:
tests refined and added